cloudflared/rootfs/etc/cont-init.d/10-cloudflared-config.sh

```diff
@@ -352,6 +360,112 @@ hasCustomConfig() {
     bashio::exit.nok "No custom config found: ${data_path}/config.yml please create custom config file or remove 'custom_config' option"
 }
 
+# ------------------------------------------------------------------------------
+# Create route for local IPs if Warp is enabled
+# ------------------------------------------------------------------------------
+createRoute() {
+    if ! bashio::config.has_value 'warp_enable' || bashio::config.false 'warp_enable' ; then
```

Comment:
See comment on bottom. I think we should move that check to the main method before calling it